packages/core/src/app/customer/Customer.tsx

```diff
@@ -117,7 +117,7 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
         this.draftEmail = email;
 
         try {
-            if (providerWithCustomCheckout !== PaymentMethodId.StripeUPE) {
+            if (providerWithCustomCheckout && providerWithCustomCheckout !== PaymentMethodId.StripeUPE) {
```

Comment:
What other providers do we have now? This looks like a bugfix then. Something should have been wrong without checking `providerWithCustomCheckout` before.

Reply:
We have stripe and bolt as `providerWithCustomCheckout`, this bit of code should only execute in the case of Bolt as far as I understand. 

cc @AlanMunozN can you please confirm?

Reply:
@animesh1987 yes, should only execute in the case of Bolt


<img width="366" alt="image" src="https://user-images.githubusercontent.com/42154828/220214143-3de8253c-5d84-4efb-a5c1-a7b73e904578.png">